client/my-sites/site-settings/navigation.jsx

```diff
@@ -96,15 +96,11 @@ export default React.createClass( {
 							</NavItem>
 					}
 
-					{
-						( ! site.jetpack && config.isEnabled( 'manage/seo' ) ||
-							site.jetpack && config.isEnabled( 'jetpack/seo-tools' ) ) &&
-						<NavItem
-							path={ `/settings/seo/${ site.slug }` }
-							selected={ section === 'seo' } >
-								{ strings.seo }
-						</NavItem>
-					}
+					<NavItem
+						path={ `/settings/seo/${ site.slug }` }
+						selected={ section === 'seo' } >
+							{ strings.seo }
```

Comment:
there formatting here seems wrong. maybe it's the hanging `>` on the line above, but I believe that this line is indented one too far